### PR TITLE
Fix 'Show Contexts' button bugs

### PR DESCRIPTION
**Pull request by tikhsuP**
This PR provides the following fixes:
1. Clicking 'Enable in Settings' opens Context settings.
2. Dimming the 'Show Contexts' button when disabled.

### CODE REVIEW

**Comment by cgorringe** [comment]
Doesn't yet dim the button when the popup opens, but does work when tapping Back from the Settings, then tapping on the dimmed 'Show Contexts' button. (still work in progress)

**Comment by tikhsuP** [comment]
Instead of calling `chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) { }` twice on lines 121 and 128, should I call it before the `if (noneSelected()) { }` statement?

File: scripts/popup.js [hunk 1]
```diff
@@ -393,23 +393,41 @@ function show_wikibooks() {
 function noContextTip() {
   chrome.storage.sync.get(['alexa', 'domaintools', 'tweets', 'wbmsummary', 'annotations', 'tagcloud'], function (event) {
     // If none of the context is selected, grey out the button and adding tip when the user hovers
-    for (const context in event) { if (event[context]) { return $('#context-screen').click(show_all_screens) } }
-    if (!$('#ctxbox').hasClass('flip-inside')) { $('#ctxbox').addClass('flip-inside') }
+    for (const context in event) { 
+      if (event[context]) { 
+        $('#contextBtn').removeAttr('disabled')
```
Comment (anishsarangi) [inline scripts/popup.js:398]: Why this is required ?

File: scripts/popup.js [hunk 2]
```diff
@@ -393,23 +393,41 @@ function show_wikibooks() {
...
+        return $('#contextBtn').click(show_all_screens) 
+      }
+    }
+    if (!$('#ctxbox').hasClass('flip-inside')) {
+      $('#ctxbox').addClass('flip-inside')
+      $('#contextBtn').attr('disabled', true)
```
Comment (anishsarangi) [inline scripts/popup.js:404]: Is this line is required ? Please tell me a situation where it is useful

**Comment by cgorringe** [comment]
I haven't studied the code too closely, but I tried it out, and it looks like it all works!

You can delete the 2 lines of code and the 'TODO' that were previously commented out. (which I may have done in the past.)

**Comment by anishsarangi** [comment]
It is working fine @cgorringe . I have tested it. :-)

File: scripts/popup.js (hunk 2 above)
Comment (tikhsuP) [reply scripts/popup.js:404]: This line is required if the user initially has no contexts selected, visits a non excluded URL and clicks on the extension icon. This line dims the 'Show Contexts' button in this case, same as the Back button. The dimmed button tells the user that no context is selected and the flip tells to select contexts in the settings.

File: scripts/popup.js (hunk 1 above)
Comment (tikhsuP) [reply scripts/popup.js:398]: This line is required if you initially have any context option checked in the context menu, go to an excluded URL (eg. newtab), and click on the extension icon. This line darkens the color of the 'Show Contexts' button if the user has some contexts selected and then visits on an excluded URL (Back button does the same). The dark 'Show Contexts' button tells the user that contexts are selected in the Context menu, and the flip tells the user that this is an excluded URL.

Comment (anishsarangi) [reply scripts/popup.js:398]: I was not aware of, when to darkens the color of the 'Show Context' button. Anyway I got an overall idea. Thank you !

File: scripts/popup.js (hunk 2 above)
Comment (anishsarangi) [reply scripts/popup.js:404]: This make sense actually. Thank you for explaining !

Comment (tikhsuP) [reply scripts/popup.js:404]: Thanks :-)

File: scripts/popup.js (hunk 1 above)
Comment (tikhsuP) [reply scripts/popup.js:398]: Thank you!! :-)

**Pull request merged**